<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigureGraphicsAdvanced</class>
 <widget class="QWidget" name="ConfigureGraphicsAdvanced">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>404</width>
    <height>321</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_1">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QGroupBox" name="groupBox_1">
       <property name="title">
        <string>Advanced Graphics Settings</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QWidget" name="gpu_accuracy_layout" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="label_gpu_accuracy">
             <property name="text">
              <string>Accuracy Level:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="gpu_accuracy">
             <item>
              <property name="text">
               <string notr="true">Normal</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string notr="true">High</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string notr="true">Extreme(very slow)</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="use_vsync">
          <property name="toolTip">
           <string>VSync prevents the screen from tearing, but some graphics cards have lower performance with VSync enabled. Keep it enabled if you don't notice a performance difference.</string>
          </property>
          <property name="text">
           <string>Use VSync (OpenGL only)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="use_asynchronous_shaders">
          <property name="toolTip">
           <string>Enables asynchronous shader compilation, which may reduce shader stutter. This feature is experimental.</string>
          </property>
          <property name="text">
           <string>Use asynchronous shader building</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="use_fast_gpu_time">
          <property name="text">
           <string>Use Fast GPU Time</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="use_caches_gc">
          <property name="toolTip">
           <string>Enables garbage collection for the GPU caches, this will try to keep VRAM within 3-4 GB by flushing the least used textures/buffers. May cause issues in a few games.</string>
          </property>
          <property name="text">
           <string>Enable GPU cache garbage collection (experimental)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="af_layout" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_1">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="af_label">
             <property name="text">
              <string>Anisotropic Filtering:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="anisotropic_filtering_combobox">
             <item>
              <property name="text">
               <string>Default</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>2x (WILL BREAK THINGS)</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>4x (WILL BREAK THINGS)</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>8x (WILL BREAK THINGS)</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>16x (WILL BREAK THINGS)</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
